docs/resources/server/user.md

```diff
@@ -1 +1,35 @@
 # User Resource
+| Key | Name/Value | Type | Description |
+| --- | ---------- | ---- | ----------- |
+| 1 | Username | **string** | The name of player |
+| 2 | Userid | **number** | The ID of player |
+| 3 | Stars | **number** | The count of stars player have |
+| 4 | Demons | **number** | The count of demons player have |
+| 8 | Creatorpoints | **number** | The count of creatorpoints player have |
+| 10 | Color1 | **number** | First color of the player use |
+| 11 | Color2 | **number** | Second color of the player use |
+| 13 | Coins | **number** | The count of coins player have |
+| 16 | accountID | **number** | The accountid of this player |
+| 17 | Usercoins | **number** | The count of usercoins player have |
+| 18 | Messagestate | **number** | 0: All, 1: Only friends, 2: None |
+| 19 | FriendsState | **number** | 0: All, 1: None |
+| 20 | Username | **string** | The name of player |
+| 21 | accIcon | **number** | The icon number of the player use |
+| 22 | accShip | **number** | The ship number of the player use |
+| 23 | accBall | **number** | The ball number of the player use |
+| 24 | accBird | **number** | The bird number of the player use |
+| 25 | accDart(wave) | **number** | The dart(wave) number of the player use |
+| 26 | accRobot | **number** | The robot number of the player use |
+| 28 | accGlow | **number** | The glow number of the player use |
+| 29 | unknown | **number** | unknown but always 1 |
+| 30 | GlobalRank | **number** | The global rank of this player |
+| 31 | Friendstate | **number** | 0: None, 1: already is friend, 3: send request to target, but target haven't accept, 4: target send request, but haven't accept 
+| 43 | accSpider | **number** | The spider number of the player use |
+| 44 | Username | **string** | The name of player |
+| 45 | Username | **string** | The name of player |
```

Comment:
key #45 is the player's twitch username

Reply:
fixed...
I found my twitter, yt, twitch is same of userName....